src/App.svelte

```diff
@@ -64,8 +90,11 @@
       }
 
       // update tax info
-      taxes = curr_taxes + socSec + medicare;
-      afterTax = intIncome - taxes;
+      taxes = curr_taxes + socSec + medicare + capitalGainsLongTermTax;
+
+      // according to ChatGPT (very reputable source ikik),
+      // LT capital gains is reported as part of your income, even though they're taxed differently
+      afterTax = (capitalGainsLongTerm - capitalGainsLongTermTax) + intIncome - taxes;
```

Comment:
I thought about it a bit more and we should probably rename `afterTax` to `takeHomeIncome` or something like that because that's basically what it becomes once you include other types of taxes. 

Also, it may be better to have 3 variables tracking capital gains taxes, income taxes + short term capital gains, and total taxes paid respectively. 

Sorry for expanding the scope, but could you rename the `afterTax` variable and split the `taxes` variable in 3 as well?